tensordict/tensordict.py

```diff
@@ -3022,8 +3022,7 @@ def set(
     ) -> TensorDictBase:
         """Sets a value in the TensorDict.
 
-        If inplace=True (default is False), and if the key already exists, set will
-        call set_ (in-place setting).
+        See TensorDictBase.set
```

Comment:
I'd rather have no docstring at all and keep the one from tensordictbase, which will be used where needed in the doc.
This can be put as a comment rather than a docstring.